arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -312,6 +312,7 @@
       "post": {
         "operationId": "DatabaseAccounts_RegenerateKey",
         "description": "Regenerates an access key for the specified Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```

Comment:
We still need the eventual return value documented - just as DatabaseAccounts_CreateOrUpdate documents the 200 response. 

---
In reply to: [87484179](https://github.com/Azure/azure-rest-api-specs/pull/702#discussion_r87484179) [](ancestors = 87484179)